### DISCUSSION TITLE
2 Questions !

### DISCUSSION THREAD

**Discussion by EvieFloof** [blueedgetechno/win11React#282] [Q&A] [answered]
I have two questions !
The first is simple : your file explorer is real ( real files ) or is it a "simulation" ?
<del>And the second Who may seem silly to you : How to launch a local copy of Windows 11 in React to modify it ?</del>

**Comment by ghost** [comment, marked as answer]
File explorer is simulated, the file explorer isn't linked to the devices.

**Comment by EvieFloof** [comment]
But I mean , is it a file system in the source code or a " json file " ?

**Reply by ghost** [reply]
Aah, for that it's stored in a JSON file, at /src/containers/applications/apps/assets/dir.json

**Comment by blueedgetechno** [comment]
And to launch your local copy,

1. clone this repo (aka download it)
2. go to the directory and open a terminal there
3. run `npm install`
4. then, run `npm start`

It is a React application (if you know about ReactJs), it will change in real time as you will change the code.

**Reply by Posandu** [reply]
or if you have git run this
```
git clone https://github.com/blueedgetechno/win11React.git && cd win11React && npm i && npm start
```

**Comment by EvieFloof** [comment]
It's late to reply , but thanks !
